packages/components/src/focal-point-picker/test/index.tsx

```diff
@@ -8,6 +8,19 @@ import userEvent from '@testing-library/user-event';
  * Internal dependencies
  */
 import Picker from '..';
+import type { FocalPointPickerProps } from '../types';
+
+type Log = { name: string; args: any[] };
+type EventLogger = ( name: string, args: any[] ) => void;
```

Comment:
Should we use `unknown` instead of `any` for the `args`?

```suggestion
type Log = { name: string; args: unknown[] };
type EventLogger = ( name: string, args: unknown[] ) => void;
```